uni-nav.env.prod

```diff
@@ -0,0 +1,9 @@
+# environment variables for the dev env
```

Comment:
These are prod env vars -- might want to update the note.

Reply:
Ok, will do after the merge 👍